kornia/enhance/adjust.py

```diff
@@ -508,6 +508,48 @@ def equalize3d(input: torch.Tensor) -> torch.Tensor:
     return torch.stack(res)
 
 
+def _invert(input: torch.Tensor) -> torch.Tensor:
+    r"""Invert color of the input 2D tensor.
+
+    Args:
+        input (torch.Tensor): image tensor with shapes like :math:(C, D, H, W) or :math:(B, C, D, H, W) to equalize.
+
+    Returns:
+        torch.Tensor: Inverted image or images.
+    """
+    _validate_input_dtype(input, accepted_dtypes=[torch.float16, torch.float32, torch.float64], min=0., max=1.)
+    return 1. - input
```

Comment:
why not adding an input variable `max_val` so that `return max_val - input` it can be `max_val: float = 1.`
/cc @ducha-aiki

Reply:
ping @shijianjian to implement this for v0.5